app/views/candidate_mailer/find_another_course.text.erb

```diff
@@ -12,8 +12,6 @@ Find out if there are any other suitable courses available:
 
 <%= I18n.t('find_postgraduate_teacher_training.production_url') %>
 
-If you find a course, sign in to your account to add the course as soon as possible:
-
-<%= candidate_magic_link(@candidate) %>
+If you find a course, [sign in to your account to add the course as soon as possible](<%= candidate_magic_link(@candidate) %>).
```

Comment:
is this email still in use? haven't seen this for aaaaages

Reply:
No, it doesn't seem to be referred to from anywhere. Will delete.